File: app/src/main/java/com/playdate/app/ui/interfaces/OnInnerFragmentClicks.java
package com.playdate.app.ui.interfaces;

import androidx.fragment.app.Fragment;

public interface OnInnerFragmentClicks {
    void ReplaceFrag(Fragment fragment);
//    void ReplaceFragWithStack(Fragment fragment);
    void NoFriends();
    void Reset();
    void loadProfile(String UserID);
    void loadMatchProfile(String UserID);
}

File: app/src/main/java/com/playdate/app/ui/interfaces/OnInnerFragmentClicks.kt
package com.playdate.app.ui.interfaces

import androidx.fragment.app.Fragment

interface OnInnerFragmentClicks {
    fun ReplaceFrag(fragment: Fragment?)
    fun ReplaceFragWithStack(fragment: Fragment?)
    fun NoFriends()
    fun Reset()
    fun loadProfile(UserID: String?)
    fun loadMatchProfile(UserID: String?)
}